src/index.js

```diff
@@ -26,28 +26,45 @@ import APIRequests from './Fetch';
 // inputs
 const userLogin = document.querySelector('.user-login');
 const userPassword = document.querySelector('.user-password');
+const dropdownCalendar = document.querySelector('.calendar');
 
 // buttons
 const loginButton = document.querySelector('.submit-login');
+const makeBookingButton = document.querySelector('.make-booking');
+const showAvailableRoomsButton = document.querySelector('.see-available-rooms-button');
+const backToChooseDateButton = document.querySelector('.back-to-choose-date-button');
+const filterByTypeDropdown = document.querySelector('.choose-by-type');
+const returnGuestHomeViewButton = document.querySelector('.return-homeview');
+const seePastBookingsButton = document.querySelector('.past-bookings');
+const seeUpcomingBookingsButton = document.querySelector('.upcoming-bookings');
 
 // page views
 const loginView = document.querySelector('.login-view');
 const navSection = document.querySelector('.nav-buttons-section');
 const guestHomeView = document.querySelector('.guest-home-view');
 const managerView = document.querySelector('.manager-view');
+const chooseDateView = document.querySelector('.guest-choose-date-view');
+const guestBookRoomView = document.querySelector('.guest-book-room-view');
+const roomIsBookedView = document.querySelector('.room-booked-view');
 
 // other elements
 const headingGuestName = document.querySelector('.heading-name');
 const loginAlert = document.querySelector('.login-alert');
-const guestViewPastBookings = document.querySelector('.cards-of-rooms');
+const guestViewBookings = document.querySelector('.cards-of-rooms'); // maybe change var name!
 const currentRoomsAvailable = document.querySelector('.list-rooms-available');
-// const headingName = document.querySelector('.heading-name');
-// const spentName = document.querySelector('.spent-name');
-// const spentAmout = document.querySelector('.spent-amount');
+const guestViewRoomCards = document.querySelector('.guest-rooms-available-by-date');
 
 // Event Listeners
```

Comment:
In the querySelectors, I need to remember to go through and find any that are only use once so they are not global variables!